jgrapht-ext/src/test/java/org/jgrapht/ext/DOTUtilsTest.java

```diff
@@ -73,7 +73,8 @@ public void testConvertGraphToDotString()
 
         //@formatter:off
         String expectedGraphAsDotString =
-                "digraph G {"            +
+                DOTExporter.DONT_ALLOW_MULTIPLE_EDGES_KEYWORD + " " +
+                  DOTExporter.DIRECTED_GRAPH_KEYWORD + " G {" +
                   "  1 [ label=\"A\" ];" +
                   "  2 [ label=\"B\" ];" +
                   "  3 [ label=\"C\" ];" +
```

Comment:
I added the 2 `DOTImporter` tests and one for missing the ID or both strict-ID. Some of the previous tests already account for the lack of strict but with ID.